Sources/ZcashLightClientKit/Modules/Service/LightWalletService.swift

```diff
@@ -142,20 +142,34 @@ struct LightWalletServiceFactory {
     }
 }
 
+enum ServiceMode: Equatable {
+    enum Constants {
+        static let rare = "rare"
+    }
+    
+    case defaultTor
+    case direct
+    case torInGroup(String)
+    
+    static func txIdGroup(prefix: String, txId: Data) -> String {
+        "\(prefix)-\(txId.hexEncodedString())"
```

Comment:
Document this. And unless there's a documented reason not to do so, this should just build the `ServiceMode`:
```suggestion
    static func txIdGroup(prefix: String, txId: Data) -> ServiceMode {
        torInGroup("\(prefix)-\(txId.hexEncodedString())")
```

Reply:
Updated